tls/s2n_async_pkey.c

```diff
@@ -409,3 +422,140 @@ S2N_RESULT s2n_async_pkey_sign_free(struct s2n_async_pkey_op *op)
 
     return S2N_RESULT_OK;
 }
+
+int s2n_async_pkey_op_get_op_type(struct s2n_async_pkey_op *op, s2n_async_pkey_op_type *type)
+{
+    POSIX_ENSURE_REF(op);
+    POSIX_ENSURE_REF(type);
+
+    switch (op->type) {
+        case S2N_ASYNC_DECRYPT:
+            *type = S2N_ASYNC_DECRYPT;
+            break;
```

Comment:
Why do you need a switch here? Can't you just do `*type = op->type`?

Reply:
Oh yes, not sure what I was doing here